bedrock/careers/templates/careers/includes/testimonials.html

```diff
@@ -5,50 +5,52 @@
 #}
 {% block careers_content %}
   <section class="c-careers-testimonials c-careers-section no-js-hidden">
-    <div class="mzp-l-content">
-      <ul class="mzp-l-content mzp-l-columns mzp-t-columns-two c-careers-testimonials-wrapper">
-        <li class="has-modal" data-modal-id="perla-d">
-          <p>
-            "Despite a diverse set of backgrounds, we share a similar passion and need for data and browser transparency..."
-          </p>
+      <ul class="mzp-l-content c-careers-testimonials-wrapper">
+        <li class="c-careers-testimonial-item has-modal" data-modal-id="perla-d">
+          <q>
+            Despite a diverse set of backgrounds, we share a similar passion...
+          </q>
           <a href="#perla-d-testimonial">
-          Read full testimonial
+          Read full Quote
           </a>
         </li>
-        <li class="has-modal" data-modal-id="lauren-n">
-          <p>
-            "Despite a diverse set of backgrounds, we share a similar passion and need for data and browser transparency..."
-          </p>
+        <li class="c-careers-testimonial-image"></li>
+        <li class="c-careers-testimonial-item has-modal" data-modal-id="lauren-n">
+          <q>
+            Despite a diverse set of backgrounds, we share a similar passion...
+          </q>
           <a href="#lauren-n-testimonial">
-          Read full testimonial
+          Read full Quote
           </a>
         </li>
-        <li class="has-modal" data-modal-id="chance-y">
-          <p>
-            "Mozilla are 100% in support of remote work and deeply values remote workers like me... "
-          </p>
+        <li class="c-careers-testimonial-image"></li>
+        <li class="c-careers-testimonial-item has-modal" data-modal-id="chance-y">
+          <q>
+            Mozilla are 100% in support of remote work and deeply values remote workers like me...
```

Comment:
**Nit(grammar):** Looks like this was a grammar error from copy's side. The full testimonial has `Mozilla is...` instead of `are...`
```suggestion
            Mozilla is 100% in support of remote work and deeply values remote workers like me...
```